samples/bluetooth/periodic_adv/src/main.c

```diff
@@ -12,7 +12,7 @@ static const struct bt_data ad[] = {
 	BT_DATA(BT_DATA_MANUFACTURER_DATA, mfg_data, 3),
 };
 
-static const struct bt_data sd[] = {
+static const struct bt_data ad_name[] = {
```

Comment:
```suggestion
static const struct bt_data ad[] = {
```

And suggest to rename the above `ad` to `per_adv_ad` or something, as it is only being used for periodic advertising